ansible_deployer/command_line.py

```diff
@@ -18,14 +18,29 @@
 
 APP_CONF = "/etc/ansible-deploy/ansible-deploy.yaml"
 CFG_PERMISSIONS = "0o644"
-SUBCOMMANDS = ("run", "list", "lock", "unlock", "verify")
+SUBCOMMANDS = ("run", "list", "lock", "unlock", "verify", "show")
 
 def verify_subcommand(command: str):
     """Function to check the first arguments for a valid subcommand"""
     if command not in SUBCOMMANDS:
         print("[CRITICAL]: Unknown subcommand :%s", (command), file=sys.stderr)
         sys.exit("55")
 
+def verify_switches(switches: list):
```

Comment:
Isn't that the job for validate_options?

Reply:
Show command seems unique so far because it's only subcommand that accepts following positional arguments. It doesn't make much difference from user perspective (miliseconds maybe) but I think positional and optional arguments should be validated separately. I propose merging verify_subcommand and verify_switches in this commit or after restructure (#84 ).